wagtail/admin/templates/wagtailadmin/account/settings_panels/avatar.html

```diff
@@ -0,0 +1,37 @@
+{% load i18n wagtailadmin_tags %}
+{% block content %}
+    <div class="avatar-panel">
+        <div class="avatar-panel__image">
+            <div class="avatar large">
+                <img src="{% avatar_url request.user 100 %}" alt="">
+            </div>
+        </div>
+        <div class="avatar-panel__form">
+            <label for="id_avatar-avatar">{% trans "Upload a profile picture:" %}</label>
+            <input type="file" name="avatar-avatar" accept="image/*" id="id_avatar-avatar">
+            <input type="hidden" name="avatar-avatar-clear" id="avatar-avatar-clear_id">
+
+            {% if form.avatar.value %}
+                <button class="avatar-panel__revert-to-gravatar button button-secondary" data-gravatar-url="{% avatar_url request.user 100 gravatar_only=True %}">{% trans "Revert to gravatar" %}</button>
```

Comment:
I think we should avoid explicitly referring to gravatars in the UI. Ever since we started supporting uploadable avatars, there's no reason for the typical user to know or care what a gravatar is - I'd consider the fact that we support gravatar at all to be a delighter / easter egg for the 1% who happened to have one previously set up. It's also possible for site owners to replace or disable gravatar support via WAGTAIL_GRAVATAR_PROVIDER_URL, which would make the label inaccurate.

Maybe "reset to default" is the simplest way of covering all bases?

Reply:
Came here to raise a point about `WAGTAIL_GRAVATAR_PROVIDER_URL` (prompted by the CSP issue) and, here it is, a true delighter from @gasman ;)

Reply:
Thanks I agree! I do think there's some value in telling users where their avatar is coming from though, could we make it say "gravatar" if they haven't changed this setting, but "default" otherwise?

Reply:
No, I don't think so - that means it's still there in the out-of-the-box setup, and presenting users with a term they don't understand and don't _need_ to understand is simply bad UX. For those users, it might as well say "frobnicate warp drive". :-)

The one case where it might be justifiable to mention Gravatar is if we detect that the user does have one, in which case it could be part of some tooltip help saying "this profile picture has been imported automatically from Gravatar" or similar, possibly with the option to drop it and fall back to the plain 'user' icon (does that option exist already?). That's probably overkill for this PR though.

Reply:
Just made this change now, let me know if there's anything else!